src/core/entities/crux-user.ts

```diff
@@ -30,6 +30,7 @@ export interface ICruxUserInformation {
 export interface ICruxUserData {
     configuration: ICruxUserConfiguration;
     privateAddresses: ICruxUserPrivateAddresses;
+    blacklistAddresses: string[];
```

Comment:
Why is it called blacklistAddresses? We're not blacklisting addresses, we're blacklisting crux IDs

Reply:
changed it to `blacklistedCruxUsers`